dokka-subprojects/plugin-base/src/main/kotlin/org/jetbrains/dokka/base/renderers/html/HtmlRenderer.kt

```diff
@@ -344,14 +347,14 @@ public open class HtmlRenderer(
                 val distinct =
                     groupDivergentInstancesWithSourceSet(it.value, it.key, pageContext,
                         beforeTransformer = { instance, _, sourceSet ->
-                            createHTML(prettyPrint = false).prepareForTemplates().div {
+                            createSmallHTML(prettyPrint = false).prepareForTemplates().div {
                                 instance.before?.let { before ->
```

Comment:
I think we can  extract  `instance.before?.let` from `createHTML` to avoid the redundant invocation of `createHTML`

Reply:
I will take a look, thanks!